// [[Reason: checker.typeToTypeNode deliberately fails on types that originate from node_modules.]] ////

//// [tests/cases/compiler/declarationEmitUsingAlternativeContainingModules1.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,5 +1,19 @@
 
+
+//// [src/index.d.mts]
+interface IEntry {
+    API: string;
+    Description: string;
+    Auth: string;
+    HTTPS: boolean;
+    Cors: string;
+    Link: string;
+    Category: string;
+}
+export declare const useEntries: () => UseQueryReturnType<IEntry[], Error>;
+export {};
+//# sourceMappingURL=index.d.mts.map
 /// [Errors] ////
 
 src/index.mts(29,31): error TS2304: Cannot find name 'UseQueryReturnType'.
 src/index.mts(29,31): error TS4025: Exported variable 'useEntries' has or is using private name 'UseQueryReturnType'.
